openrtb_ext/request.go

```diff
@@ -4,7 +4,15 @@ import "encoding/json"
 
 // ExtRequest defines the contract for bidrequest.ext
 type ExtRequest struct {
-	Prebid ExtRequestPrebid `json:"prebid"`
+	Prebid   ExtRequestPrebid   `json:"prebid"`
+	Currency ExtRequestCurrency `json:"currency"`
+}
+
+// ExtRequestCurrency defines the contract for bidrequest.ext.currency
+type ExtRequestCurrency struct {
+	AdServerCurrency      string          `json:"adServerCurrency,omitempty"`
+	GranularityMultiplier float64         `json:"granularityMultiplier,omitempty"`
+	Rates                 json.RawMessage `json:"rates,omitempty"`
```

Comment:
Any reason you don't define the real type here? Types help make the code self-documenting.

Reply:
ah yes no reason. updating.